xc7/fasm2bels/verilog_modeling.py

```diff
@@ -1265,9 +1265,8 @@ def make_iosettings_map(self, parsed_eblif):
         )
 
         # Regex for matching ports belonging to a single inout port
-        INOUT_RE = re.compile(
-            r"(.*)(_\$inp$|_\$inp(\[[0-9]+\])$|_\$out$|_\$out(\[[0-9]+\])$)(.*)"
-        )
+        INOUT_IN_RE = re.compile("(.*)_\$inp(.*)")
```

Comment:
I'd restrict the second part of those regex to `\[[0-9]+\]` so it matches only numbers in brackets instead of arbitrary text. Off course it also has to have a case when there is nothing there.

Reply:
After the `$out` or `$inp` there can be only the signal position in the bus information or nothing right?

E.g.:
- Can be: `signal_$inp[10]`
- Can be: `signal_$inp`
- Cannot be: `signal_$inp_other_stuff`

Reply:
It should not be, but `signal_$inp_other_stuff` is also a legal net name. I want to avoid leaving a potential bug in the code that may hit us in the future.

Reply:
Ok, makes sense. I'll revert the change then, as separating in IN and OUT is not making the regex simpler.

Reply:
Done